website/docs/blueprints/inference/inference-charts.md

```diff
@@ -0,0 +1,248 @@
+---
+sidebar_label: Inference Charts
+---
+
+# AI on EKS Inference Charts
+
+The AI on EKS Inference Charts provide a streamlined Helm-based approach to deploy AI/ML inference workloads on both GPU
+and AWS Neuron (Inferentia) hardware. This chart supports multiple deployment configurations and comes with
+pre-configured values for popular models.
+
+## Overview
+
+The inference charts support the following deployment types:
+
+- **GPU-based VLLM deployments** - Single-node VLLM inference
+- **GPU-based Ray-VLLM deployments** - Distributed VLLM inference with Ray
+- **Neuron-based VLLM deployments** - VLLM inference on AWS Inferentia chips
+- **Neuron-based Ray-VLLM deployments** - Distributed VLLM inference with Ray on Inferentia
+
+## Prerequisites
+
+Before deploying the inference charts, ensure you have:
+
+- Kubernetes cluster with GPU or AWS Neuron nodes
+- Helm 3.0+
+- For GPU deployments: NVIDIA device plugin installed
+- For Neuron deployments: AWS Neuron device plugin installed
+- Hugging Face Hub token (stored as a Kubernetes secret)
+
+## Quick Start
+
+### 1. Create Hugging Face Token Secret
+
+Create a Kubernetes secret with your Hugging Face token:
+
+```bash
+kubectl create secret generic hf-token --from-literal=token=your_huggingface_token
+```
+
+### 2. Deploy a Pre-configured Model
+
+Choose from the available pre-configured models and deploy:
+
+```bash
```

Comment:
I would add a caution here to ensure users have access to these GPU or Neuron instances before trying the cost implications